### ISSUE TITLE
VB tuples: Interface implementation with array of tuples not detected

### ISSUE THREAD

**Issue by jcouv** [dotnet/roslyn#13303]
See `TupleArrays`

```VB
        <Fact(Skip:="REMOVE")>
        Public Sub TupleArrays()

            Dim verifier = CompileAndVerify(
<compilation>
    <file name="a.vb">
Interface I
    Function M(a As (Integer, Integer)()) As System.ValueTuple(Of Integer, Integer)()
End Interface

Class C
    Implements I

    Shared Sub Main()
        Dim i As I = new C()
        Dim r = i.M(new System.ValueTuple(Of Integer, Integer)() { new System.ValueTuple(Of Integer, Integer)(1, 2) }
        System.Console.Write($"{r(0).Item1} {r(0).Item2}")
    End Sub

    Public Function M(a As (Integer, Integer)()) As System.ValueTuple(Of Integer, Integer)()
        Return New System.ValueTuple(Of Integer, Integer)() { (a(0).Item1, a(0).Item2) }
    End Function
End Class

    </file>
</compilation>, additionalRefs:=s_valueTupleRefs, expectedOutput:=<![CDATA[2]]>)

            ' REMOVE AFTER REVIEW
            ' Getting error: a.vb(6) : error BC30149: Class 'C' must implement 'Function M(a As (Integer, Integer)()) As (Integer, Integer)()' for interface 'I'.

            verifier.VerifyDiagnostics()

        End Sub
```

**Comment by jcouv** [comment]
I suspected it was a tuple unification issue, but @VSadov pointed out that VB needs `Implements I.M`.
Vlad, I'll let you close this issue once you check in the updated test. Thanks!

**Comment by jcouv** [comment]
The updated test was checked in and looks good.
